### PR TITLE
💬  Breach screen - remove Start date and display Status date

### PR DESCRIPTION
**Pull request by sarahyjja**

### CODE REVIEW

**Comment by sarahyjja** [comment]
I mismatch the tickets 😬 This one is 830 https://dsdmoj.atlassian.net/browse/PIC-830

File: server/views/case-summary-record-order-breach.njk [hunk 1]
```diff
@@ -73,6 +69,10 @@
                     <dt class="govuk-summary-list__key">Status</dt>
                     <dd class="govuk-summary-list__value">{{ breach.status }}</dd>
                 </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">Status date</dt>
+                    <dd class="govuk-summary-list__value">{{ moment(breach.started, 'YYYY-MM-DD').format(displayDateFormat) }}</dd>
```
Comment (theDustRoom) [inline server/views/case-summary-record-order-breach.njk:74]: This needs to be `breach.statusDate`

**Review by theDustRoom — CHANGES_REQUESTED** [review]

**Comment by sarahyjja** [comment]
I think this ticket has been a bit confusing for me with PIC-831.
The two issues are solved here so I will relate the other ticket to here.
😬

File: server/views/case-summary-record-order.njk [hunk 2]
```diff
@@ -221,6 +222,7 @@
         </div>
     </div>
 
+    {% set breachData = [] %}
```
Comment (katie-cawthorne) [inline server/views/case-summary-record-order.njk:225]: Same here

File: server/views/case-summary-record-order.njk [hunk 3]
```diff
@@ -21,6 +21,7 @@
 } %}
 
 {%- set communityData = data.communityData -%}
+{%- set breachData = data.communityData.breach %}
```
Comment (katie-cawthorne) [inline server/views/case-summary-record-order.njk:24]: Variable breachData has been created but unused

**Review by katie-cawthorne — COMMENTED** [review]
All good just the comments about the unused breachData variables

File: server/views/case-summary-record-order.njk (hunk 3 above)
Comment (sarahyjja) [reply server/views/case-summary-record-order.njk:24]: delete it 👍

File: server/views/case-summary-record-order.njk (hunk 2 above)
Comment (sarahyjja) [reply server/views/case-summary-record-order.njk:225]: delete it 👍

**Review by katie-cawthorne — APPROVED** [review]
Nice one ✨

**Review by katie-cawthorne — APPROVED** [review]
Nice one ✨

**Review by theDustRoom — APPROVED** [review]

**Pull request merged**